Description: skip requests lib tests, if urllib3 is too old
Author: Dimitri John Ledkov <xnox@ubuntu.com>

Index: python-wsgi-intercept/test/test_requests.py
===================================================================
--- python-wsgi-intercept.orig/test/test_requests.py	2014-07-30 17:41:45.000000000 +0800
+++ python-wsgi-intercept/test/test_requests.py	2014-07-31 14:47:46.000000000 +0800
@@ -1,5 +1,10 @@
 import py.test
-from wsgi_intercept import requests_intercept, WSGIAppError
+
+# Requests interceptor requires 1.8 urllib3.
+import urllib3
+pytestmark = py.test.mark.skipif(hasattr(urllib3, 'connection') == False,
+                                 reason="requires urllib3 1.8+")
+
 from test import wsgi_app
 from test.install import installer_class
 import requests
@@ -7,8 +12,12 @@
 
 HOST = 'some_hopefully_nonexistant_domain'
 
-InstalledApp = installer_class(requests_intercept)
-
+try:
+    from wsgi_intercept import requests_intercept, WSGIAppError
+    InstalledApp = installer_class(requests_intercept)
+except ImportError:
+    if hasattr(urllib3, 'connection'):
+        raise
 
 def test_http():
     with InstalledApp(wsgi_app.simple_app, host=HOST, port=80) as app:
